config/locales/social_share_button.zh-CN.yml

```diff
@@ -0,0 +1,19 @@
+'zh-CN':
```

Comment:
You don't need this file unless you're going to surprise us all and implement a Chinese language :grin: